test/compute-engine/canonical-form.test.ts

```diff
@@ -342,15 +393,41 @@ describe('CANONICAL FORMS', () => {
         canonForms = PositiveInfinity
         canonical  = PositiveInfinity
       `);
-      expect(checkPower('{-a/4}^1')).toMatchInlineSnapshot(`
-        box        = ["Power", ["Divide", ["Negate", "a"], 4], 1]
-        canonForms = ["Divide", ["Negate", "a"], 4]
-        canonical  = ["Multiply", ["Rational", -1, 4], "a"]
-      `);
-      expect(checkPower('{-\\pi}^1')).toMatchInlineSnapshot(`
-        box        = ["Power", ["Negate", "Pi"], 1]
-        canonForms = ["Negate", "Pi"]
-        canonical  = ["Negate", "Pi"]
+
+      //@note: This correctly simplifies, unlike would be expected of 'x^1' (where x is a
+      //user-declared numeric constant with a default 'holdUntil' value of 'evaluate'), because 'Pi'
+      //is a library-defined constant, consequently always being canonical & bound (and ∴ having a
+      //type/value), even in *non-canonical* exprs.
+      //^!Note that currently, 'x^1' *will simplify* at present, but this would not be considered
+      //correct behaviour (compute-engine/pull/238#discussion_r2034335185). This is because
+      //canonicalization of symbols (including partial-canonicalization) is presently, erroneously,
+      //synonymous with 'binding': permitting its type/value to be determined at this stage.
+      expect(checkPower('{\\pi}^1')).toMatchInlineSnapshot(`
+        box        = ["Power", "Pi", 1]
+        canonForms = Pi
+        canonical  = Pi
+      `);
+
+      //↓🙁: cannot be simplified, at least via the route of checking the 'type' of the base/'-\pi',
+      //because is a non-canonical function-expr. (non-bound to 'Power' definition)
+
+      // expect(checkPower('{-\\pi}^1')).toMatchInlineSnapshot(`
+      //   box        = ["Power", "Pi", 1]
+      //   canonForms = Pi
+      //   canonical  = Pi
+      // `);
+
+      /*
+       * Control
+       */
+      //!@note:
+      //Not currently simplified, despite 'j' being an 'integer'-typed constant, because the base is
+      //a non-canonical function and therefore unbound to a definition; consequently with its 'type'
+      //non-determinable.
+      expect(checkPower('{-j/4}^1')).toMatchInlineSnapshot(`
+        box        = ["Power", ["Divide", ["Negate", "j"], 4], 1]
+        canonForms = ["Power", ["Divide", ["Negate", "j"], 4], 1]
+        canonical  = ["Multiply", ["Rational", -1, 4], "j"]
```

Comment:
A similar case to above

Reply:
Yes, this might need to be addressed by introducing the concept of **global scopes**...